src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -99,10 +106,12 @@ function pipeDrupalPagesIntoMetalsmith(contentData, files) {
     data: {
       nodeQuery: { entities: pages },
       sidebarQuery: sidebarNav,
+      facilitySidebarQuery: facilitySidebarNav,
```

Comment:
@ncksllvn @jbalboni this breaks the build if it is not an `object`: if the menu is missing from the drupal content (`null` in drupal.json). how can we check if the `facilitySidebarQuery`/`sidebarQuery` is/are not null while destructuring?

Reply:
Do you mean that sometimes `sidebarQuery` is `null`? Would it help at all to assign a default value?

```
sidebarQuery: sidebarNav = {}, // prevent null value
```

Sorry if I'm misunderstanding!